ds_judgements_public_ui/sass/includes/_footer.scss

```diff
@@ -82,3 +82,16 @@
     font-size: $typography-md-text-size;
   }
 }
+
+.more-info-footer {
```

Comment:
As this isn't within the footer and instead inside the `recent_judgments` component, can we move this CSS to that component and prefix with the appropriate class i.e. `recent-judgments`

Reply:
updated